dear_petition/petition/utils.py

```diff
@@ -0,0 +1,19 @@
+from django.utils.timezone import make_aware
+import pytz
+from datetime import datetime, date
+
+
+def dt_obj_to_date(dt_obj):
+    """converts a datetime obj to a date"""
+    return dt_obj.date()
+
+
+def make_datetime_aware(dt_str):
+    """makes datetime string an aware datetime object (UTC ==> America/New York)"""
+    if dt_str is None or dt_str == "":
+        return None
+    datetime_obj = datetime.strptime(dt_str, "%Y-%m-%dT%H:%M:%S")
```

Comment:
Since we're using the `"%Y-%m-%dT%H:%M:%S"` in a bunch of places, could we define it as a constant in the settings?

Reply:
added `DATETIME_FORMAT` to constants